### PR TITLE
Update messages.zh.yml

### PR DESCRIPTION
**Pull request by FLYEMOJ1**
Updated few "Machine Translate" make it feels more like "Human Translated"

### CODE REVIEW

**Comment by FLYEMOJ1** [comment]
**least 2-3 commspt member approve and let who created this merge to merge to master!!**

**Review by Paper-white — APPROVED** [review]

File: translations/messages.zh.yml [hunk 1]
```diff
@@ -789,7 +789,7 @@
 "Configure your imap settings which will be used to fetch emails from your mailbox.": "配置您的imap设置，该设置将用于从您的邮箱中获取电子邮件。"
 "Transport": "传输"
 "IMAP": "地图"
-"Gmail": "邮箱"
+"Gmail": "谷歌邮箱"
```
Comment (SerinaNya) [inline translations/messages.zh.yml]: 还是 `Gmail` 比较好

**Review by SerinaNya — CHANGES_REQUESTED** [review]

File: translations/messages.zh.yml [hunk 2]
```diff
@@ -1523,5 +1523,5 @@
 "Internal server error": "内部服务器错误"
 "Our system has goofed up for a while, but good part is it won't last long": "我们的系统已经有一段时间了，但好的部分是它不会持续很长时间"
 "Unknown Error": "未知错误"
-"We are quite confused about how did you land here:/": "我们很困惑你是怎么在这里着陆的 :/"
+"We are quite confused about how did you land here:/": "我们很困惑你是怎么在打开这个页面的 :/"
```
Comment (gxres042) [inline translations/messages.zh.yml]: 我们很困惑你是如何打开这个页面的

**Review by gxres042 — COMMENTED** [review]
看看这样改会不会更合适

**Review by SerinaNya — APPROVED** [review]

**Review by gxres042 — APPROVED** [review]

**Comment by FLYEMOJ1** [comment]
**3 of review are approved.**
Merge to master.

**Pull request merged**